versioned_docs/version-V3/guides/providing-liquidity/decrease-liquidity.md

```diff
@@ -0,0 +1,68 @@
+---
+id: decrease-liquidity
+title: Decrease Liquidity
+sidebar_position: 4
+---
+
+Make sure to go through the [first guide](./setting-up-your-contract.md) before continuing to this section
+
+Here we decrease the liquidity of our position, without withdrawing all of it. This example assume the contract already has possession of the NFT, and requires that the calling address is the owner of that NFT.
```

Comment:
requires that the calling address is the original owner of that NFT (since the owner is now technically the contract as it has possession of the NFT)